stable/node-local-dns/values.yaml

```diff
@@ -16,7 +16,7 @@ config:
   # Use a custom upstreamsvc for -upstreamsvc
   customUpstreamsvc: ""
 
-  # If true, it will bind 0.0.0.0, otherwise dnsServer and localDns will be used. https://github.com/bottlerocket-os/bottlerocket/issues/3711#issuecomment-1907087528
```

Comment:
@Sarga you wrote this, care to comment? Is this change valid?

Reply:
@max-rocket-internet @andrii29 
The changes are correct, it was my mistake. Thanks a million for noticing and correcting it!